client/stylesheets/_toCat.scss

```diff
@@ -4346,12 +4349,55 @@ On click the images will be displayed at normal size to complete the effect
     margin-left: 5px;
     vertical-align: middle;
     color: #fff !important;
+} */
+
+.sortClicker i {
+	margin-left: 8px;
+	margin-bottom: -2px;
+	color: #fff !important;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used<br>Line should be indented with spaces, not tabs